debian/Dockerfile

```diff
@@ -1,11 +1,16 @@
-ARG PHP_IMAGE_TAG=7.2-fpm-stretch
-FROM php:${PHP_IMAGE_TAG}
+ARG PHP_VERSION=7.2
+ARG INVOICENINJA_VERSION
+
+FROM php:${PHP_VERSION}-fpm-stretch
 
 LABEL maintainer="Samuel Laulhau <sam@lalop.co>"
 
 #####
 # SYSTEM REQUIREMENT
 #####
+COPY ./alpine/entrypoint.sh /usr/local/bin/docker-entrypoint
```

Comment:
is that normal you copy the alpine entrypoint ?

Reply:
Good catch. Fixed.